src/content/ui-api/management-api/crd/awsmachinedeployments.infrastructure.giantswarm.io.md

```diff
@@ -565,7 +565,7 @@ spec:
 </div>
 
 <div class="annotation-description">
-<p>This annotation is used for configuring the subnet size of AWSCluster or AWSMachineDeployment. The value is a number that will represent the subnet mask used when creating the subnet. This value must be smaller than 28 due to AWS restrictions.</p>
+<p>This annotation is used to configure the workload cluster's node pool subnet size when used on an AWSMachineDeployment resource. The value is a number that will represent the subnet mask used when creating the subnet. It must be smaller than 28 due to AWS restrictions.</p>
```

Comment:
Adding formatting to a string literal.

```suggestion
<p>This annotation is used to configure the workload cluster's node pool subnet size when used on an `AWSMachineDeployment` resource. The value is a number that will represent the subnet mask used when creating the subnet. It must be smaller than 28 due to AWS restrictions.</p>
```

If that should be missing in other CRD names here, please apply accordingly. Thanks!